### DISCUSSION TITLE
Playlist: Configure a schedule to autoswitch between playlists on shared monitor

### DISCUSSION THREAD

**Discussion by neshorg** [grafana/grafana#53057] [Dashboards, Panels & Visualizations]
Recently I was thinking that it might be useful to be able to create a few playlists and let them play in a defined order based upon a schedule. 
For example, you would create a playlist group (e.g. "Playlist Group - Dev Team"). 
In this group you're able to place a few playlists (e.g. "Playlist A", "Playlist B" & "Playlist C") on which you're able to order them by defining a schedule for each one:

- Playlist Group - Dev Team
1. Playlist A, scheduled between 8:00 AM and 12:00 PM
|- dashboard1
|- dashboard2
|- dashboard3
3. Playlist B, scheduled between 12:00 PM and 18:00 PM
|- dashboard1
|- dashboard3
|- dashboard4
|- dashboard5
5. Playlist C, scheduled between 18:00 PM and 8:00 AM
|- dashboard1
|- dashboard6

With this, the "Dev Team" would automatically have their playlist change on the monitor accordingly during the day depending on the current team and night shifts which would require different set of playlists. The night shift team would never require certain dashboards from the playlist in the morning (playlist A) or after noon (playlist B), thus needing a more efficient and short spanned playlist custom made for them while making use of the same monitor in the office. 

This was an example of course, but in my case I have to change the playlist myself due to one part of the office sharing 1 monitor with different teams during the day vs during the night.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
